recipes/sfml/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+cmake -D CMAKE_BUILD_TYPE=Release                 \
+      -D CMAKE_INSTALL_PREFIX=$PREFIX             \
+      -D UDEV_PATH_INCLUDES=$PREFIX/include       \
+      -D UDEV_PATH_LIB=$PREFIX/lib                \
+      -D CMAKE_INSTALL_LIBDIR=lib                 \
+      -D CMAKE_PREFIX_PATH=$PREFIX                \
```

Comment:
This should not be needed as it is set by https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/50d5eca739d954e69798f0c4b3b56319a7922236/recipe/activate-gcc.sh#L98